addons/csi-aws-ebs/csi-snapshot-webhook.yaml

```diff
@@ -0,0 +1,153 @@
+# Source: https://github.com/kubernetes-csi/external-snapshotter/blob/v6.1.0/deploy/kubernetes/webhook-example/rbac-snapshot-webhook.yaml
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: snapshot-webhook
+  namespace: kube-system
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: snapshot-webhook-runner
+rules:
+  - apiGroups: ["snapshot.storage.k8s.io"]
+    resources: ["volumesnapshotclasses"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["groupsnapshot.storage.k8s.io"]
+    resources: ["volumegroupsnapshotclasses"]
+    verbs: ["get", "list", "watch"]
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: snapshot-webhook-role
+subjects:
+  - kind: ServiceAccount
+    name: snapshot-webhook
+    namespace: kube-system
+roleRef:
+  kind: ClusterRole
+  name: snapshot-webhook-runner
+  apiGroup: rbac.authorization.k8s.io
+
+# Source: https://github.com/kubernetes-csi/external-snapshotter/blob/v6.3.3/deploy/kubernetes/webhook-example/webhook.yaml
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: snapshot-validation-secret
+  namespace: kube-system
+data:
+  "cert.pem": |
+{{ .Certificates.CSIWebhookCert | b64enc | indent 4 }}
+  "key.pem": |
+{{ .Certificates.CSIWebhookKey | b64enc | indent 4 }}
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: "validation-webhook.snapshot.storage.k8s.io"
+  namespace: kube-system
+webhooks:
+- name: "validation-webhook.snapshot.storage.k8s.io"
+  rules:
+  - apiGroups:   ["snapshot.storage.k8s.io"]
+    apiVersions: ["v1"]
+    operations:  ["CREATE", "UPDATE"]
+    resources:   ["volumesnapshots", "volumesnapshotcontents", "volumesnapshotclasses"]
+    scope:       "*"
+  clientConfig:
+    service:
+      namespace: kube-system
+      name: "snapshot-validation-service"
+      path: "/volumesnapshot"
+    caBundle: |
+{{ .Certificates.KubernetesCA | b64enc | indent 6 }}
+  admissionReviewVersions: ["v1"]
+  sideEffects: None
+  failurePolicy: Ignore # We recommend switching to Fail only after successful installation of the webhook server and webhook.
```

Comment:
We need to discuss this.

Reply:
@xmudrii this is changed from previously "Fail", question is, should we change it back to the Fail or leave it along?

Reply:
@kron4eg I don't have strong opinions, maybe reverting it to Fail is a safer approach